src/main/java/ch/naviqore/service/impl/PublicTransitServiceImpl.java

```diff
@@ -41,12 +42,18 @@ public class PublicTransitServiceImpl implements PublicTransitService {
 
     // TODO: Make CutOff configurable, if walk duration is longer than this, no first or last walk is needed
     private static final int MIN_WALK_DURATION = 120;
+    // do not make the cache configurable, since this solution will be replaced by trip masking in the raptor
```

Comment:
Even if we go with masks, we will then probably cache masks so still good to keep. If this has to be configurable is of course debatable.